config/environments/production.rb

```diff
@@ -19,9 +19,12 @@
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
   # config.require_master_key = true
 
-  # Disable serving static files from the `/public` folder by default since
-  # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  # Unless explicitly disabled, serve static files via puma. This has been the
+  # default for Dockerfile for a very long time already.
+  config.public_file_server.enabled = (ENV['RAILS_SERVE_STATIC_FILES'] || "true") == "true"
```

Comment:
I am not completely sure about this, also if we do change the default, what about `ENV.fetch('RAILS_SERVE_STATIC_FILES', 'true')`?